### ISSUE TITLE
`w`, `d`, and `q` incorrectly parsed as opcodes

### ISSUE THREAD

**Issue by io12** [skeeto/x86-lookup#10]
One of the x86 manual headers (page 1673/2242 on the combined section 2 PDF) says the following:

```
VPBROADCASTB/W/D/Q—Load with Broadcast Integer Data from General Purpose Register
```

Trying to lookup `w`, `d`, or `q` jumps to this header. My guess is that the parser incorrectly treats the text between the slashes as separate instructions.

**Comment by skeeto** [comment]
Yup, you're right. That's not intended and will need to be fixed. Thanks 
for reporting it.
